tests/watch_test.go

```diff
@@ -15,7 +15,7 @@ func TestWatch(t *testing.T) {
 	publisher := getLocalConnection()
 	subscriber := getLocalConnection()
 
-	rp := fireCommandAndGetRESPParser(subscriber, "SUBSCRIBE k1")
+	rp := fireCommandAndGetRESPParser(subscriber, "SUBSCRIBE SELECT k1")
 	if rp == nil {
 		t.Fail()
```

Comment:
Should this be changed to `QWATCH` instead of `SUBSCRIBE` and even the file name? Let's work towards making this more closer to the final version.

Reply:
Done.